shared/src/business/entities/cases/Case.ts

```diff
@@ -109,7 +108,7 @@ export class Case extends JoiValidationEntity {
   public closedDate?: string;
   public createdAt: string;
   public docketNumber: string;
-  public docketNumberSuffix?: string;
+  public docketNumberSuffix?: string | null;
```

Comment:
Forgive me if I'm misunderstanding: would this be handled by the mapper? I can see why it would be undefined, but not null

Reply:
The issue is here:

https://github.com/ustaxcourt/ef-cms/blob/fb8372c8911b56c7e086dc308dda5fb04950c8cf/shared/src/business/utilities/getDocketNumberSuffix.ts#L14

Probably should just return undefined, but I think that broke something and I said, "Ugh, let's deal with that later and just be honest here for now."